sidebars.js

```diff
@@ -26,8 +26,9 @@ const sidebars = {
       items: [
         "learn/prediction-markets",
         "learn/liquidity",
-        "learn/market-rules",
         "learn/using-zeitgeist-markets",
+        "learn/betting-strategy",
+        "learn/market-rules",
```

Comment:
Why changing the order here? Do we need to know about the betting strategy before the market rules?

Reply:
I'm putting the educational stuff together, followed by more "administrative" things.